lib/standard_deviation.rb

```diff
@@ -1,5 +1,56 @@
-ages = [24, 30, 18, 20, 41]
+# Super Sports Games
+require 'pry'
 
-# Your code here for calculating the standard deviation
+class SportsGames
+  def initialize
+    @ages = [24, 30, 18, 20, 41]
+  end
 
-# When you find the standard deviation, print it out
+  def sum_of_integers
+    sum  = 0
+    @ages.each do |number|
+      sum += number
+      binding.pry
+    end
+    sum
+  end
+
+  def number_of_integers
+    @ages.count
+  end
+
+  def average
```

Comment:
We will be going over scope on Monday, but inside this method, ruby can not access the variable `sum`